client/src/components/pages/common/errors/Error-404/index.css

```diff
@@ -0,0 +1,21 @@
+*{
+  margin: 0;
+}
+.error_404{
+  width: 100%; 
+  height: 99vh; 
+} 
```

Comment:
image need to be fixed and take margin from the top ,  also if we removed the app bar , I think it will be better .